### ISSUE TITLE
Reduce zombie gray hands?

### ISSUE THREAD

**Issue by admaciaszek** [jantic/DeOldify#198]
The colorization works very well for faces but when it comes hands it very often not recognized and left completely gray which distracts from the rest of the wonderfully render frame. Would using higher quality photos help or hurt like if I used upscaling for the video? Also would render factor affect the amount of zombie hands my gpu can handle up to factor 39? Should I use a different .pth in models?

**Comment by jantic** [comment]
Higher quality images help a lot, in general.  You have to be careful though- a lot of what is done artificially to increase the quality of an image- noise reduction for example- often helps but sometimes winds up throwing off the model and making things worse.  Why?  Because a lot of what the model bases its colorization on is textures and minute patterns, and this upscaling/denoising processes often introduce subtle differences that aren't quite natural that the model just isn't use to seeing.

Similarly, increasing render_factor, as a rule of thumb, will get the details better (like determining what is skin), but that rule of thumb is highly dependent on input quality.  If the input quality is low, then you're actually better off with low render_factor.

Basically your best bet is to use the rule of thumb that yes- higher quality inputs and higher render_factors are going to do better in general, but ultimately you have to experiment to get best results.  Luckily there's the handy feature in the notebooks and Colabs to do just that- under the title "See how well render_factor values perform on a frame here" at the botom.  The idea there is that you can choose a frame and see which render_factor will work best for that frame. Generally- if it works best for that frame, it's probably a good bet for the rest of the scene/video.
